packages/cli/src/commands/generate/resource/index.ts

```diff
@@ -0,0 +1,143 @@
+
+    // uppercase first letter
+    const resource = uppercaseFirstChar(resourceName);
+
+    // get the project type
+    const projectType = getProjectType();
+    const uiFramework = getUIFramework();
+
+    const sourceDir = `${__dirname}/../templates/resource/${projectType}`;
+
+    // create temp dir
+    const tempDir = generateTempDir();
+
+    // copy template files
+    copySync(sourceDir, tempDir);
+
+    const compileParams = {
+        resourceName,
+        resource,
+        actions: customActions || defaultActions,
+        projectType,
+        uiFramework,
+    };
+
+    // compile dir
+    compileDir(tempDir, compileParams);
+
+    // delete ignored actions
+    if (customActions) {
+        defaultActions.forEach((action) => {
+            if (!customActions.includes(action)) {
+                unlinkSync(`${tempDir}/${action}.tsx`);
+            }
+        });
+    }
+
+    const { alias } = getResourcePath(projectType);
+
+    // copy to destination
+    const resourceFolderName = plural(resourceName).toLowerCase();
+    const destinationResourcePath = `${options.path}/${resourceFolderName}`;
+
+    let moveSyncOptions = {};
+
+    // empty dir override
+    if (!pathExists(destinationResourcePath)) {
+        moveSyncOptions = { overwrite: true };
+    }
+    moveSync(tempDir, destinationResourcePath, moveSyncOptions);
+
+    // clear temp dir
+    temp.cleanupSync();
+
+    const jscodeshiftExecutable = require.resolve(".bin/jscodeshift");
+    const { stderr, stdout } = execa.sync(jscodeshiftExecutable, [
+        "./",
+        "--extensions=ts,tsx,js,jsx",
+        "--parser=tsx",
+        `--transform=${__dirname}/../src/transformers/resource.ts`,
+        `--ignore-pattern=**/.cache/**`,
+        `--ignore-pattern=**/node_modules/**`,
+        `--ignore-pattern=**/build/**`,
+        `--ignore-pattern=**/.next/**`,
+        // pass custom params to transformer file
+        `--__actions=${compileParams.actions}`,
+        `--__pathAlias=${alias}`,
+        `--__resourceFolderName=${resourceFolderName}`,
+        `--__resource=${resource}`,
+        `--__resourceName=${resourceName}`,
+    ]);
+
+    // console.log(stdout);
+
+    if (stderr) {
+        console.log(stderr);
+    }
+
+    console.log(
+        `Resource (${destinationResourcePath}) generated successfully! 🎉`,
+    );
+};
+
+const generateTempDir = (): string => {
+    temp.track();
+    return temp.mkdirSync("resource");
+};
+
+const getResourcePath = (
+    projectType: ProjectTypes,
+): { path: string; alias: string } => {
+    switch (projectType) {
+        case ProjectTypes.NEXTJS:
+            return {
+                path: "src/components",
+                alias: "@components",
+            };
+        case ProjectTypes.REMIX:
+            return {
+                path: "app/components",
+                alias: "~/components",
+            };
+    }
+
+    // vite and react
+    return {
+        path: "src/pages",
+        alias: "pages",
```

Comment:
```suggestion
        alias: "pages",
```
May cause errors in projects not created by superplate